### ISSUE TITLE
Support Bytes on SQLite

### ISSUE THREAD

**Issue by internalfx** [prisma/prisma#6666]
## Problem

Data model mapping for SQLite does not support Bytes.

## Suggested solution

Add Bytes support for SQLite.

## Additional context

I would argue that storing files in the database makes more sense for SQLite than any other database. SQLite even proposes itself as a replacement for `fopen()`.

```
SQLite reads and writes small blobs (for example, thumbnail images) 35% faster¹ than the same blobs can be read from or written to individual files on disk using fread() or fwrite().
```
https://sqlite.org/fasterthanfs.html

**Comment by Brendonovich** [comment]
Prisma already supports Bytes with SQLite, it was just missing from the documentation for a while. 

@janpio I'm guessing you forgot to close this issue when you closed #10537?

**Comment by janpio** [comment]
Indeed 👍
